config/application.rb

```diff
@@ -46,6 +46,7 @@
 ENV['MYSQL_HOST'] ||= "mysql"
 ENV['MYSQL_PORT'] ||= "3306"
 ENV['TRUSTED_IP'] ||= "10.0.40.1"
+ENV['LEVRIERO_URL'] ||= "https://api.datacite.org"
```

Comment:
Is this going to work?

Reply:
Well, actually here is where I get to the POST method discussion in https://api.datacite.org. If I make a POST request in a consolidated API, then it will write to the database, all the time. unless we use admin domain for ES:POST, right?